keyboards/lets_split/keymaps/poker/keymap.c

```diff
@@ -1,98 +1,95 @@
 #include QMK_KEYBOARD_H
 
-
-extern keymap_config_t keymap_config;
-
 // Each layer gets a name for readability, which is then used in the keymap matrix below.
 // The underscores don't mean anything - you can have a layer called STUFF or any other name.
 // Layer names don't all need to be of the same length, obviously, and you can also skip them
 // entirely and just use numbers.
-#define _QWERTY 0
-#define _COLEMAK 1
-#define _DVORAK 2
-#define _LOWER 3
-#define _RAISE 4
-#define _FN 5
-#define _ADJUST 6
+enum layer_names {
+    _QWERTY,
+    _COLEMAK,
+    _DVORAK,
+    _LOWER,
+    _RAISE,
+    _FN,
+    _ADJUST
```

Comment:
just a nit. 
```suggestion
    _ADJUST,
```

The comma at the end prevents formatting from condensing to one line

Reply:
Updated!

I prefer trailing commas anyway. They weren't there before so I didn't add them.